src/store/migrations/14_rsk_derivation_path.js

```diff
@@ -0,0 +1,75 @@
+import { getDerivationPath } from '@/utils/derivationPath'
+import { Networks } from '@/utils/networks'
+import { accountCreator, getNextAccountColor } from '@/utils/accounts'
+import { ChainId, chains } from '@liquality/cryptoassets'
+
+export const rskDerivationPath = {
```

Comment:
Name of migration doesn't seem appropriate
shoudl be more like `account_set_derivation_path`

Reply:
got it, fixed